dapp/js/containers/Pages/CampaignDeployer/Component.jsx

```diff
@@ -28,6 +28,16 @@ export default class Deployer extends Component {
 
   componentDidMount() {
     this.props.setAccount(web3.eth.accounts[0]);
+    //set default account to user current account if they are not set
+    let campaignValues = Object.assign({}, this.props.campaignValues);
+    let accountTypes = ['escapeCaller', 'securityGuard', 'donor', 'recipient']
+    for(let i=0; i<accountTypes.length; i++) {
+        if(!campaignValues[accountTypes[i]]){
+            campaignValues[accountTypes[i]] = web3.eth.accounts[0];
+        }
+    }
+    this.props.updateCampaignValues(campaignValues);
```

Comment:
Yes, i think there are something to be refactorized. however, as i noticed Griff said that you are starting to work on a new app? so these changes here we are talking about are just temporarily right